modules/ROOT/pages/producing-ign.adoc

```diff
@@ -130,6 +130,23 @@ curl https://getfedora.org/static/fedora.gpg | gpg --import
 gpg --verify butane-x86_64-apple-darwin.asc
 ----
 
+==== Windows
```

Comment:
Should this say e.g. `Windows (using PowerShell)`?

Reply:
The Linux and macOS standalone procedures don't specify that this should be run in a shell and I assume the intended audience for Windows understands that as well. `cmd.exe` might be a contender and source of confusion, though, maybe? Hovering over the code snippet and the actual syntax should put these doubts to rest, though.